.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
Not sure why your `.gitignore` didn't ignore all files under `.idea/`. 
Your team will decide whether we should remove this file or not. Personally I would remove this.

Reply:
Removed.